src/Abstractions/TaskOrchestrationContext.cs

```diff
@@ -385,7 +385,7 @@ public virtual Task CallSubOrchestratorAsync(TaskName orchestratorName, TaskOpti
     /// </summary>
     /// <param name="categoryName">The logger's category name.</param>
     /// <returns>An instance of <see cref="ILogger"/> that is replay-safe.</returns>
-    public ILogger CreateReplaySafeLogger(string categoryName)
+    public virtual ILogger CreateReplaySafeLogger(string categoryName)
```

Comment:
If you still want this change, it is fine to leave in. Otherwise if you think `Moq.Protected` is sufficient, we can remove this.

Reply:
I'd like to keep this as I'd like for customer to be able to mock this directly :)